### ISSUE TITLE
mouseovers in hpcviewer don't always display long function names

### ISSUE THREAD

**Issue by jmellorcrummey** [HPCToolkit/hpcviewer#90]
there seems to be a problem with long names that don't contain spaces. sometimes, it brings up the popop, which flashes and disappears. other times, the popup doesn't appear.

I can't share the database where this occurs (adagio). We will need to make a separate test case.

**Comment by laksono** [comment]
I think I know what the problem is. The wrapper is looking for space to wrap the function name.  See https://github.com/HPCToolkit/hpcviewer/blob/a44f6701445f2bab89a733b098ff248f00ed7e00/edu.rice.cs.hpc.data/src/edu/rice/cs/hpc/data/util/string/StringUtil.java#L24

I'll look for a better algorithm for text wrapper. I think Apache's [WordUtils](https://commons.apache.org/proper/commons-text/javadocs/api-release/org/apache/commons/text/WordUtils.html) is a better candidate as it can wrap long words.
https://commons.apache.org/proper/commons-text/javadocs/api-release/org/apache/commons/text/WordUtils.html#wrap-java.lang.String-int-java.lang.String-boolean-

**Comment by jmellorcrummey** [comment]
I agree. That will forcefully wrap long words, even if there aren't spaces.
